decentraland/unity-renderer
# Feat: added UI blur and world blur resources

## What does this PR change?

Fix #2183 
Added blur resources and applied blur effect on top header subsection in explore v2

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/ui-blur
2. Open the explore v2
3. Go into backpack
4. Verify that the top section of the explore header is blurring the background

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
